packages/core/src/Machine.ts

```diff
@@ -26,7 +26,7 @@ export function Machine<
   config: MachineConfig<TContext, TStateSchema, TEvent>,
   options?: Partial<MachineOptions<TContext, TEvent>>,
   initialContext?: TContext
-): StateMachine<TContext, TStateSchema, TEvent>;
+): StateMachine<TContext, TStateSchema, TEvent, any>;
```

Comment:
Do you know why this `any` is needed? I see the default value for this type parameter being `Typestate<TContext>` and I don't immediately see how this helps anything here. I still learn TS so maybe I'm missing something obvious~

Reply:
If a Typestate isn't provided, we need to default to `any`.

Reply:
Could you elaborate? When the default applies then? Do we **always** need to supply `any`? Or is this situation here different somehow from other ones?

Reply:
`any` no longer needs to be supplied.